features/features_test.go

```diff
@@ -0,0 +1,19 @@
+package features
+
+import (
+	"testing"
+
+	"github.com/letsencrypt/boulder/test"
+)
+
+func TestFeatures(t *testing.T) {
```

Comment:
Would you be open to adding a few more test cases?

I don't think `Export()` and `Reset()` have any tests and I noticed there's nothing testing a non-existent feature given to `Enabled()` will return an error.